packages/nextjs/app/builders/0xD2692F9df925D18D527ABe8b3d99EE9E9C8d75AE/_components/SkillComponent.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
If you're using Next.js, you no longer need to import React manually, as Next.js automatically includes it for you

Reply:
@NightmareFox12 don't forget to push the changes :)